diff -r 36cb94a16edf config.mk
--- a/config.mk Tue Jul 08 20:02:47 2014 +0100
+++ b/config.mk Mon Apr 27 08:53:45 2020 -0700
@@ -13,8 +13,8 @@
 XINERAMAFLAGS = -DXINERAMA

 # Xft, comment if you don't want it
-XFTINC = -I/usr/include/freetype2
-XFTLIBS  = -lXft -lXrender -lfreetype -lz -lfontconfig
+# XFTINC = -I/usr/include/freetype2
+# XFTLIBS  = -lXft -lXrender -lfreetype -lz -lfontconfig

 # includes and libs
 INCS = -I${X11INC} ${XFTINC}
