diff --git a/bin/git-fuzzy b/bin/git-fuzzy
index a7b2b19..544f9c7 100755
--- a/bin/git-fuzzy
+++ b/bin/git-fuzzy
@@ -47,30 +47,11 @@ lib_dir="/usr/share/git-fuzzy"
 
 # -- Invariants --
 
-if ! type git >/dev/null 2>&1; then
-  # shellcheck disable=2016
-  gf_log_error '`git` not found; it is required for `git fuzzy` to work.'
-  exit 1
-fi
-
 if ! gf_is_in_git_repo; then
   gf_log_error "not in git repo"
   exit 1
 fi
 
-if ! type fzf >/dev/null 2>&1; then
-  # shellcheck disable=2016
-  gf_log_error '`fzf` not found; it is required for `git fuzzy` to work.'
-  exit 1
-fi
-
-FZF_VERSION="$(fzf --version)"
-MIN_FZF_VERSION="0.20.0"
-if [ "$FZF_VERSION" = "$(echo -e "$FZF_VERSION\n$MIN_FZF_VERSION" | sort -V | head -n1)" ]; then
-  # shellcheck disable=2016
-  gf_log_warning '`fzf` is too old and may not work properly'
-fi
-
 # NB: checking for actually _unset_
 # shellcheck disable=2016
 if [ -z "${HUB_AVAILABLE+X}" ]; then
